.github/workflows/ci.yml

```diff
@@ -58,6 +58,19 @@ jobs:
           git --version
           node --version
           npm --version
+          npx typescript --version
+      - name: Upgrade typescript
+        run: npm install -g typescript@4.9.5
```

Comment:
@realmarv this is not exactly what I mean when I recommended upgrading typescript; I didn't say "upgrade typescript explicitly", I meant to have latest typescript in the same way we had it before I committed the downgrade to commitlint17.0.0: by not having any package*.json files at all.

Reply:
Done